data/members.yml

```diff
@@ -2,7 +2,7 @@ types:
   - name: members
     items:
     -
-      image: /images/edgenative/members/adlink.svg
+      image: /images/edgenative/members/logo-a_d_l_i_n_k.svg
```

Comment:
Why is ADLINK spelled a_d_l_i_n_k? This seems like an odd choice.

Reply:
This is to prevent AdBlock from blocking the browser from downloading the image.
On Chrome, when the file is called `adlink.svg` it gets blocked by adblock because of its name (containing `ad`)

https://www.keycdn.com/support/how-to-solve-err-blocked-by-client:
```
Sometimes when checking your Chrome browser's console you may have noticed an error such as _net::ERR_BLOCKED_BYCLIENT appearing for a particular site you are visiting or working on. This error is commonly caused due to one of the extensions installed within Chrome. Most likely you will see this error appear if you are using an extension such as Adblock or a browser safety plugin.
```

https://stackoverflow.com/questions/17255493/how-to-stop-adblock-plus-blocking-images-in-html-page:
```
Adblock will block images

with common ad dimensions in the file name, e.g. 'myimage_720_90.png' or 'myimage_300x250.jpg'.
with common ad keywords in their path (and file name), e.g. banners, adv, ad.
maybe other filters that analyse CSS or HTML
```

Reply:
We it this way, because the way it was spelled before "adlink" was causing ad blockers in e.g. Chrome to block the image! This logo was never visible in all browsers, but is after the fix.

Reply:
Great catch! Thanks for the update!